feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list.component.html

```diff
@@ -32,6 +32,7 @@ <h3 class="cx-ticketing-list-title">
           (viewPageEvent)="pageChange($event)"
         ></cx-pagination>
       </div>
+      <cx-customer-ticketing-create></cx-customer-ticketing-create>
```

Comment:
Would this not be shown if there's no tickets?

Reply:
Yes, we fixed this in the dummy-branch but forgot to add it here.

Reply:
I don't think this was resolved

Reply:
I will add it as SubTask for @adolfobee